Cargo.toml

```diff
@@ -42,9 +42,6 @@ inline_asm = []
 abi_x86_interrupt = []
 const_fn = []
 
-[package.metadata.docs.rs]
-rustdoc-args = ["--cfg", "docsrs"]
-
```

Comment:
We still need to pass `--cfg docsrs` when building for [docs.rs](https://docs.rs). This is so we can enable the `doc_cfg` feature, allowing us to note which features are required for which functions. For example:

With `docsrs`:
![with](https://user-images.githubusercontent.com/5506060/127803959-d2c15f77-a17d-49c9-a32c-8346c971ed2e.png)

Without `docsrs`:
![without](https://user-images.githubusercontent.com/5506060/127803953-c490d5c5-49bc-4a7a-818c-ef98b98e65c8.png)

Reply:
I just decided to add a `doc_cfg` nightly-only feature to handle this. Now running `cargo doc` just does the right thing by default.